susemanager-utils/susemanager-sls/src/modules/uyuni_config.py

```diff
@@ -0,0 +1,1247 @@
+
+        :param channel_label: label of the channel
+        :param uid: user login id
+        :param access: Flag which if user should have management access to channel or not
+        :return: 1 on success, exception thrown otherwise.
+        """
+        log.debug("change managing access to %s for user %s in channel %s", access, uid, channel_label)
+        return self.client("channel.software.setUserManageable", channel_label, uid, access)
+
+    def set_user_subscribable(self, channel_label: str, uid: str, access: bool) -> int:
+        """
+        Set the subscribable flag for a given channel and user.
+        If value is set to 'true', this method will give the user subscribe permissions to the channel.
+        Otherwise, that privilege is revoked.
+
+        :param channel_label: label of the channel
+        :param uid: user login id
+        :param access: Flag which if user should subscribe a channel or not
+        :return: 1 on success, exception thrown otherwise.
+        """
+        log.debug("change subscription access to %s for user %s in channel %s", access, uid, channel_label)
+        return self.client("channel.software.setUserSubscribable", channel_label, uid, access)
+
+    def is_user_manageable(self, channel_label: str, uid: str) -> bool:
+        """
+        Returns whether the channel may be managed by the given user.
+
+        :param channel_label: label of the channel
+        :param uid: user login id
+        :return: boolean which indicates if user can manage channel or not
+        """
+        log.debug("check if user %s can manage channel %s", uid, channel_label)
+        return bool(self.client("channel.software.isUserManageable", channel_label, uid))
+
+    def is_user_subscribable(self, channel_label: str, uid: str) -> bool:
+        """
+        Returns whether the channel may be subscribed to by the given user.
+
+        :param channel_label: label of the channel
+        :param uid: user login id
+        :return: boolean which indicates if user subscribe the channel or not
+        """
+        log.debug("check if user %s can subscribe channel %s", uid, channel_label)
+        return bool(self.client("channel.software.isUserSubscribable", channel_label, uid))
+
+    def is_globally_subscribable(self, channel_label: str) -> bool:
+        """
+        Returns whether the channel is globally subscribed on the organization
+        :param channel_label: label of the channel
+        :return: boolean which indicates if channel is globally subscribe
+        """
+        log.debug("check if channel globally Subscribable %s", channel_label)
+        return bool(self.client("channel.software.isGloballySubscribable", channel_label))
+
+
+class UyuniOrg(UyuniRemoteObject):
+    """
+    CRUD operations on orgs
+    """
+
+    def list_orgs(self) -> Dict[str, Union[int, str, bool]]:
+        """
+        List all orgs.
+        Admin user must have SUSE Manager Administrator role to perform this action
```

Comment:
I would suggest

```
    Note: the configured admin user must have the SUSE Manager/Uyuni Administrator role to perform this action
```

This line is also present with minor variants in many other places.